### ISSUE TITLE
[a11y][SideNavigationItem]: Tooltip announced different in Chrome and Safari

### ISSUE THREAD

**Issue by ArtemJJS** [SAP/ui5-webcomponents#11141]
### Bug Description

Using VoiceOver as a screen reader, there is a difference in the way how SideNavigationItem is read in Chrome and Safari when adding a tooltip.
In Safari SideNavigationItem.text attribute is not read, but in Chrome is read. That makes correct usage of the tooltip attribute for providing cross-browser a11y support impossible.
Doesn't matter will SideNavigationItem text be read or not, but it is expected that the behavior is the same for different browsers.
Environment: MacOs laptop + VoiceOver + Chrome + Safari

### Affected Component

SideNavigationItem

### Expected Behaviour

In both, Chrome and Safari, SideNavigationItem is read in the same way by VoiceOver screen reader.

### Steps to Reproduce

1. Create SideNavigation with SideNavigationItem
2. Add tooltip with any text to SideNavigationItem
3. Access SideNavigationItem with VoiceOver

### Log Output, Stack Trace or Screenshots

Safari:
https://github.com/user-attachments/assets/e8d183bc-f98d-4050-9612-bc4806bc54cd

Chrome:
https://github.com/user-attachments/assets/9a6dc75d-2149-4b43-b0e7-21a38ab8223b

### Priority

None

### UI5 Web Components Version

2.7.0

### Browser

Safari

### Operating System

MacOS

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Valkanister** [comment]
Hello @SAP/ui5-webcomponents-topic-rd,
 I cannot check this on Windows, but please have a look at the SideNavigationItem's tooltip announcement in Voiceover.

Kind regards, 
Georgi

**Comment by LidiyaGeorgieva** [comment]
Hello @ArtemJJS ,

Please give more detailed description of the issue.
The text in  the videos are changing very fast and I'm not sure what is missing.
I prepared this [very simple example](https://sap.github.io/ui5-webcomponents/nightly/play/#eyJpbmRleC5odG1sIjp7Im5hbWUiOiJpbmRleC5odG1sIiwiY29udGVudCI6IjwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuPCFET0NUWVBFIGh0bWw-XG48aHRtbCBsYW5nPVwiZW5cIj5cblxuXG5cbjxoZWFkPlxuICAgIFxuICAgIDxzdHlsZT5cbiAgICAgICo6bm90KDpkZWZpbmVkKSB7XG4gICAgICAgIGRpc3BsYXk6IG5vbmU7XG4gICAgICB9XG5cbiAgICBodG1sIHtcbiAgICAgIGZvcmNlZC1jb2xvci1hZGp1c3Q6IG5vbmU7XG4gICAgfVxuICAgIDwvc3R5bGU-XG5cbiAgICBcbiAgICA8c3R5bGU-XG4gICAgICAqOm5vdCg6ZGVmaW5lZCkge1xuICAgICAgICBkaXNwbGF5OiBub25lO1xuICAgICAgfVxuXG4gICAgaHRtbCB7XG4gICAgICBmb3JjZWQtY29sb3ItYWRqdXN0OiBub25lO1xuICAgIH1cbiAgICA8L3N0eWxlPlxuXG4gICAgPG1ldGEgY2hhcnNldD1cIlVURi04XCI-XG4gICAgPG1ldGEgbmFtZT1cInZpZXdwb3J0XCIgY29udGVudD1cIndpZHRoPWRldmljZS13aWR0aCwgaW5pdGlhbC1zY2FsZT0xLjBcIj5cbiAgICA8dGl0bGU-U2FtcGxlPC90aXRsZT5cbiAgICA8bGluayByZWw9XCJzdHlsZXNoZWV0XCIgaHJlZj1cIi4vbWFpbi5jc3NcIj5cbjwvaGVhZD5cblxuPGJvZHkgc3R5bGU9XCJiYWNrZ3JvdW5kLWNvbG9yOiB2YXIoLS1zYXBCYWNrZ3JvdW5kQ29sb3IpXCI-XG4gICAgPCEtLSBwbGF5Z3JvdW5kLWZvbGQtZW5kIC0tPlxuXG4gICAgPHVpNS1zaWRlLW5hdmlnYXRpb24gY29sbGFwc2VkPlxuXHRcdDx1aTUtc2lkZS1uYXZpZ2F0aW9uLWl0ZW0gdGV4dD1cIkhvbWVcImljb249XCJob21lXCIgdG9vbHRpcD1cInRvb2x0aXAgMTIzXCI-PC91aTUtc2lkZS1uYXZpZ2F0aW9uLWl0ZW0-XG5cdFx0PHVpNS1zaWRlLW5hdmlnYXRpb24taXRlbSBzbG90PVwiZml4ZWRJdGVtc1wiIHRleHQ9XCJIaXN0b3J5XCIgaWNvbj1cImhpc3RvcnlcIiB0b29sdGlwPVwidG9vbHRpcCA0NTZcIj48L3VpNS1zaWRlLW5hdmlnYXRpb24taXRlbT5cbiAgICA8L3VpNS1zaWRlLW5hdmlnYXRpb24-XG4gICAgPCEtLSBwbGF5Z3JvdW5kLWZvbGQgLS0-XG4gICAgPHNjcmlwdCB0eXBlPVwibW9kdWxlXCIgc3JjPVwibWFpbi5qc1wiPjwvc2NyaXB0PlxuPC9ib2R5PlxuXG48L2h0bWw-XG48IS0tIHBsYXlncm91bmQtZm9sZC1lbmQgLS0-XG4ifSwibWFpbi5qcyI6eyJuYW1lIjoibWFpbi5qcyIsImNvbnRlbnQiOiIvKiBwbGF5Z3JvdW5kLWhpZGUgKi9cbmltcG9ydCBcIi4vcGxheWdyb3VuZC1zdXBwb3J0LmpzXCI7XG4vKiBwbGF5Z3JvdW5kLWhpZGUtZW5kICovXG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMtZmlvcmkvZGlzdC9TaWRlTmF2aWdhdGlvbi5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWZpb3JpL2Rpc3QvU2lkZU5hdmlnYXRpb25JdGVtLmpzXCI7XG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMtZmlvcmkvZGlzdC9TaWRlTmF2aWdhdGlvbkdyb3VwLmpzXCI7XG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMtZmlvcmkvZGlzdC9TaWRlTmF2aWdhdGlvblN1Ykl0ZW0uanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy1maW9yaS9kaXN0L1NpZGVOYXZpZ2F0aW9uR3JvdXAuanNcIjtcblxuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWljb25zL2Rpc3QvaG9tZS5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWljb25zL2Rpc3QvY2hhaW4tbGluay5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWljb25zL2Rpc3QvZ3JvdXAuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy1pY29ucy9kaXN0L2xvY2F0ZS1tZS5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWljb25zL2Rpc3QvY2FsZW5kYXIuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy1pY29ucy9kaXN0L2hpc3RvcnkuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy1pY29ucy9kaXN0L2N1c3RvbWVyLmpzXCI7In0sIm1haW4uY3NzIjp7Im5hbWUiOiJtYWluLmNzcyIsImNvbnRlbnQiOiJ1aTUtc2lkZS1uYXZpZ2F0aW9uIHtcbiAgICBoZWlnaHQ6IDYwMHB4O1xufSJ9fQ) of SideNavigation. Please test it and describe what is not correct.
In addition I'm confused, because you mention text and tooltip as the same, but SideNavigationItem.text is the visualized text of the item (see https://sap.github.io/ui5-webcomponents/nightly/components/fiori/SideNavigationItem/#text) and the tooltip is used for the collapsed SideNavigation (as it is described in the documentation https://sap.github.io/ui5-webcomponents/nightly/components/fiori/SideNavigationItem/#tooltip) and you are testing on expanded SideNavigation.

Best Regards,
Lidiya

**Comment by ArtemJJS** [comment]
Hi @LidiyaGeorgieva 
I reproduced an issue based on your example by making SideNavigation uncollapsed.
Attaching additional screen records from Chrome and Safari.
So this is a transcription how item is read:
- Chrome: "History, tooltip 456"
- Safari: "tooltip 456"

Chrome screen record:
https://github.com/user-attachments/assets/1ef2cd0e-839d-4240-ab68-44fc1a77bb1d
Safari screen record
https://github.com/user-attachments/assets/01e97900-e470-4669-bfd3-1f17f80f0fdc

**Comment by GerganaKremenska** [comment]
Hello @ArtemJJS,

Thank you for reporting your issue. We do not have influence over how a screen reader interprets the provided HTML with ARIA attributes. We ensure that UI5 web components follow the ARIA standards. If a screen reader interprets the same HTML in different ways depending on the browser, this indicates an inconsistency either in the browser or the screen reader.

Closing the issue since there is no action on development side.

Best Regards,
Gergana
